discord/bot.py

```diff
@@ -1374,7 +1373,7 @@ class Bot(BotBase, Client):
     anything that you can do with a :class:`discord.Client` you can do with
     this bot.
 
-    This class also subclasses :class:`.ApplicationCommandMixin` to provide the functionality
+    This class also subclasses ``ApplicationCommandMixin`` to provide the functionality
```

Comment:
What is the reason behind this?

Reply:
It's a broken link on the docs because `ApplicationCommandMixin` isn't documented anywhere, so I converted it to a normal code block.

Reply:
Can we document it instead